test/membrane_live/accounts_test.exs

```diff
@@ -21,12 +21,16 @@ defmodule MembraneLive.AccountsTest do
     end
 
     test "create_user/1 with valid data creates a user" do
-      valid_attrs = %{email: "some email", name: "some name", picture: "some picture"}
+      valid_attrs = %{
+        email: "john@gmail.com",
+        name: "John Kowalski",
+        picture: "kowalski.img"
+      }
 
       assert {:ok, %User{} = user} = Accounts.create_user(valid_attrs)
-      assert user.email == "some email"
-      assert user.name == "some name"
-      assert user.picture == "some picture"
+      assert user.email == "john@gmail.com"
```

Comment:
I think we need an additional test for checking whether an email constraint works correctly.

Reply:
Will do 👌